packages/oc-template-react-compiler/lib/viewTemplate.js

```diff
@@ -9,11 +9,12 @@ const viewTemplate = ({
   var staticPath = model.reactComponent.props._staticPath;
   var props = JSON.stringify(model.reactComponent.props);
   return '<div id="${reactRoot}" class="${reactRoot}">' + modelHTML + '</div>' +
+    '<style>${css}</style>' +
     '<script>' +
     'window.oc = window.oc || {};' +
     'oc.cmd = oc.cmd || [];' +
     'oc.cmd.push(function(oc){' +
-      'oc.addStylesToHead(\\'${css}\\');' +
+      'oc.events.fire(\\'oc:cssDidMount\\', \\'${css}\\');' +
```

Comment:
I think we had an issue for when (in dev mode) the css has comments. Have we addressed that here? I don't see any sanitisation for that string.

Reply:
I've checked by running dev and I've added a snapshot for the string passed in the client-side acceptance test. So wasn't able yet to reproduce it, let's publish this as 5.0.0 and if @themoch will report that this still breaks in the case of comments within the css, then we can try to nail it further.

Reply:
I can pull this branch and test locally if that helps before a merge.

Reply:
That would be great @themoch

Reply:
tested locally looks good!